api/v1beta1/appwrapper_types.go

```diff
@@ -56,6 +56,9 @@ type SchedulingSpec struct {
 	Requeuing RequeuingSpec `json:"requeuing,omitempty"`
 
 	NotImplemented_DispatchDuration NotImplemented_DispatchDurationSpec `json:"dispatchDuration,omitempty"`
+
```

Comment:
I added to the spec, because that is what was done on the multi-cluster branch.

An alternative might be to just use an annotation; if all we envision ourself doing is adding a single string for a cluster name, then an annotation would be adequate.  If we think we will need more information in the future, then having a structured addition to the spec probably makes sense.

Reply:
after discussion, going with a simple label for now.